clusterdock/topologies/cdh/actions.py

```diff
@@ -77,13 +100,26 @@ def start(args):
                             **{'volumes_from': [java_image]} if args.java else {})
                        for hostname in args.secondary_nodes]
 
+    if args.kerberos:
+        kdc_node = Node(hostname=KDC_HOST_NAME, network=args.network,
+                        image=kdc_node_image, volumes=[{KERBEROS_VOLUME_DIR: KERBEROS_VOLUME_DIR}])
+
     secondary_node_group = NodeGroup(name='secondary', nodes=secondary_nodes)
     node_groups = [NodeGroup(name='primary', nodes=[primary_node]),
                    secondary_node_group]
+    if args.kerberos:
+        node_groups.append(NodeGroup(name=KDC_HOST_NAME, nodes=[kdc_node]))
```

Comment:
I'd just make the name `'kdc'` instead of the `KDC_HOST_NAME` so that this doesn't break if someone decides to update the topology some day to give the KDC node a different hostname.